clients/drcachesim/tests/memref_gen.h

```diff
@@ -84,7 +93,18 @@ gen_branch_encoded(memref_tid_t tid, addr_t pc, int encoding)
     memref.instr.encoding_is_new = true;
     return memref;
 }
+
 #elif defined(X86_64) || defined(X86_32)
+inline memref_t
+gen_instr_encoded(addr_t pc, const std::vector<char> &encoding, memref_tid_t tid = 1)
+{
+    memref_t memref = gen_instr_type(TRACE_TYPE_INSTR, tid, pc, encoding.size());
+    memref.instr.size = encoding.size();
```

Comment:
Redundant assignment. It's already set in gen_instr_type.

Reply:
Done.